images/base/Dockerfile

```diff
@@ -90,15 +90,18 @@ RUN echo "Enabling kubelet and containerd services ... " \
 RUN echo "Ensuring /etc/kubernetes/manifests" \
     && mkdir -p /etc/kubernetes/manifests
 
-RUN echo "Adjusting systemd-tmpfiles timer" \
-    && sed -i /usr/lib/systemd/system/systemd-tmpfiles-clean.timer -e 's#OnBootSec=.*#OnBootSec=1min#'
```

Comment:
the timer value is in a different location and already reasonable in the debian image